xml_converter/generators/code_generator.py

```diff
@@ -208,7 +201,7 @@ class Generator:
     def delete_generated_docs(self, dir_path: str) -> None:
         for filepath in os.listdir(dir_path):
             filepath = os.path.join(dir_path, filepath)
-            if filepath.endswith("_gen.hpp") or filepath.endswith("_gen.cpp") or filepath.endswith("_gen.html"):
+            if filepath.endswith("_gen.hpp") or filepath.endswith("_gen.cpp") or filepath.endswith(".html"):
```

Comment:
What the rational for removing "_gen" from a generated file?

Reply:
Responded in the other comment to answer the same question.